src/models.py

```diff
@@ -46,14 +48,18 @@ def __init__(self, architecture_config, training_config, callbacks_config):
         self.weight_regularization = weight_regularization_unet
         self.optimizer = optim.Adam(self.weight_regularization(self.model, **architecture_config['regularizer_params']),
                                     **architecture_config['optimizer_params'])
-        self.loss_function = [('mask', multiclass_segmentation_loss, 1.0)]
+        dice_loss = partial(multiclass_dice_loss,
+                            excluded_classes=[0])
+        loss_function = partial(mixed_dice_cross_entropy_loss,
```

Comment:
@taraspiotr the `loss_function` is implemented as a list of tuples to be able to pass multiple losses (and weigh them). I know in mapping there were other things involved and for simplicity we went with one function. Nonetheless I think this is a perfect example where one can (and I believe should) use the designed interface for passing weights and losses

Reply:
@jakubczakon The same point as @apyskir, I believe multiple losses expects multiple outputs and targets, while we use only one multichannel output.

Reply:
@taraspiotr correct. nvmd